lib/Convos/Core/Conversation.pm

```diff
@@ -9,6 +9,7 @@ has name     => sub { Carp::confess('name required in constructor') };
 has password => '';
 has topic    => '';
 has unread   => 0;
+has notifications => 0;
```

Comment:
Is it easier to read `notifications` or `n_notifications` ?

Reply:
Either works - I otherwise pondered "notifications_count" .. to me its somewhat obvious from the default being 0 .. else we'd need to change all the "unread" attributes too.

Reply:
It's obvious here, but not so obvious other places in the code where it is used. Going to resolve this issue though, since "unread" needs changing as well.